collectd-openstack/README.md

```diff
@@ -0,0 +1,300 @@
+
+  - **Memory Utilization**: Shows the percentage of utilized memory in the tenant/project.
+
+    [<img src='./img/chart-openstack-tenant-memory-util.png' width=200px>](./img/chart-openstack-tenant-memory-util.png)
+
+  - **Disk Utilization**: Shows the percentage of utilized disk in the tenant/project.
+
+    [<img src='./img/chart-openstack-tenant-disk-util.png' width=200px>](./img/chart-openstack-tenant-disk-util.png)
+
+  - **Instances Available vs Used**: Shows the number of available instances and used instances in the tenant/project.
+
+    [<img src='./img/chart-openstack-tenant-instances.png' width=200px>](./img/chart-openstack-tenant-instances.png)
+
+  - **Memory Usage**: Shows the used memory over available in the tenant/project.
+
+    [<img src='./img/chart-openstack-tenant-memory-usage.png' width=200px>](./img/chart-openstack-tenant-memory-usage.png)
+
+  - **Disk Usage**: Shows the used disk over available in the tenant/project.
+
+    [<img src='./img/chart-openstack-tenant-disk-usage.png' width=200px>](./img/chart-openstack-tenant-disk-usage.png)
+
+  - **Volumes Available vs Used**: Shows the used volumes over available volumes in the tenant/project.
+
+    [<img src='./img/chart-openstack-tenant-volumes.png' width=200px>](./img/chart-openstack-tenant-volumes.png)
+
+  - **VCPUs Available vs Used**: Shows the used virtaul cpus over available in the tenant/project.
+
+    [<img src='./img/chart-openstack-tenant-vcpus.png' width=200px>](./img/chart-openstack-tenant-vcpus.png)
+
+  - **Top Instances by CPU %**: Shows the top five instances by CPU usage percentage in the tenant/project.
+
+    [<img src='./img/chart-openstack-tenant-top-cpu.png' width=200px>](./img/chart-openstack-tenant-top-cpu.png)
+
+  - **Top Instances by Memoery Used**: Shows the top five instances by memory usage in the tenant/project.
+
+    [<img src='./img/chart-openstack-tenant-top-memory.png' width=200px>](./img/chart-openstack-tenant-top-memory.png)
+
+  - **Top Instances by VCPUs**: Shows the top five instances by virtual CPUs used in the tenant/project.
+
+    [<img src='./img/chart-openstack-tenant-top-vcpus.png' width=200px>](./img/chart-openstack-tenant-top-vcpus.png)
+
+
+- **NEUTRON**:
+
+  - **Networks**: Shows the total number of networks created in all prjects.
+
+    [<img src='./img/chart-openstack-neutron-networks.png' width=200px>](./img/chart-openstack-neutron-networks.png)
+
+  - **Routers**: Shows the total number of routers created in all prjects.
+
+    [<img src='./img/chart-openstack-neutron-routers.png' width=200px>](./img/chart-openstack-neutron-routers.png)
+
+  - **Subnets**: Shows the total number of subnets created in all prjects.
+
+    [<img src='./img/chart-openstack-neutron-subnets.png' width=200px>](./img/chart-openstack-neutron-subnets.png)
+
+  - **Flaoting IPs Available vs Used**: Shows the total number of floating IPs used over available in all prjects.
```

Comment:
another instance of "used over available"

Reply:
Updated the metric description accordingly